src/Tribe/Editor/Configuration.php

```diff
@@ -39,6 +40,14 @@ public function editor_config( $editor_config ) {
 			]
 		);
 
+		$blocks = empty( $editor_config['blocks'] ) ? [] : $editor_config['blocks'];
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 18 spaces but found 1 space

<sub>hash: 68d393df09fb6ad436577ae4f1ea129a</sub>